boxsdk/pagination/page.py

```diff
@@ -38,7 +39,7 @@ def _translator(self):
 
         :rtype:   :class:`Translator`
         """
-        return self._session.translator
+        return Translator()
```

Comment:
Why is it okay/necessary to use a new Translator() here? Maybe a comment could help?